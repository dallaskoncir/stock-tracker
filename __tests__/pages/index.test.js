import { render, waitFor } from '@testing-library/react';

import Home from '../../pages/index';

describe('<Home />', () => {
    test('renders correctly', async () => {
        const { container } = render(<Home />);

        await waitFor(() => expect(container).toMatchSnapshot());
    });
});
