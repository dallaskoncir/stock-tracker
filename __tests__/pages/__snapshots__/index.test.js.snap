// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders correctly 1`] = `
<div>
  <div
    class="rnc__base "
  >
    <div
      class="rnc__notification-container--top-full"
    />
    <div
      class="rnc__notification-container--bottom-full"
    />
    <div
      class="rnc__notification-container--top-left"
    />
    <div
      class="rnc__notification-container--top-right"
    />
    <div
      class="rnc__notification-container--bottom-left"
    />
    <div
      class="rnc__notification-container--bottom-right"
    />
    <div
      class="rnc__notification-container--top-center"
    />
    <div
      class="rnc__notification-container--center"
    >
      <div
        class="rnc__util--flex-center"
      />
    </div>
    <div
      class="rnc__notification-container--bottom-center"
    />
  </div>
  <main>
    <header>
      <h1>
        Stock Comparison
      </h1>
      <p>
        Search and select up to 3 stocks to compare.
      </p>
    </header>
    <section />
  </main>
</div>
`;
